spec/examples/with_transaction_composite_command.rb

```diff
@@ -0,0 +1,20 @@
+class WithTransactionCompositeCommand
```

Comment:
If we decide to include the helper inside `CompositeCommand` example we could remove it.